lib/core/src/Cardano/Wallet.hs

```diff
@@ -1541,8 +1541,17 @@ normalizeDelegationAddress s addr = do
 -- | A 'PartialTx' is an an unbalanced 'SealedTx' along with the necessary
 -- information to balance it.
 --
--- The 'inputs' and 'redeemers' must match the binary transaction contained in
--- the 'sealedTx'.
+-- The 'inputs' must match the binary transaction contained in
+-- the 'tx'.
```

Comment:
Question: should the sets of inputs be identical?

Reply:
Currently: yes - or there'd be undefined behaviour.

With #3255, the `PartialTx.input` would be of type `UTxO`, and as long as `ptxExtraUTxO <> walletUTxO` is able to resolve all inputs in the tx, we're good. (Meaning: superfluous entries are allowed, and omitting entries which are already in the wallet utxo is ok too)

Reply:
Perhaps the following wording would make the status quo clearer:

The set of 'inputs' must be identical to the set of inputs within the binary transaction contained in the 'tx'.